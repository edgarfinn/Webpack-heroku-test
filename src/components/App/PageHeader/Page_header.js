import React from 'react';
import './Page_header.scss';

export default () => {
  return (
    <header className="page__header">
      <i className="header__icon fa fa-github-square" aria-hidden="true"></i>
      <h1 className="header__title">Github Repo Finder</h1>
    </header>
  )
}
