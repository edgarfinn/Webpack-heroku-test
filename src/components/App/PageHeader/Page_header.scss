@import '../../../Sass/variables';


.page__header {
  background-color: $white;
  padding: 1rem;
}

.header__icon {
  display: inline;
  font-size: 2.5rem;
  float: left;
  color: $primary;
  line-height: 2rem;
}

.header__title {
  display: inline;
  font: $font-stack;
  font-size: 1.6rem;
  line-height: 2rem;
  font-weight: bold;
  color: $primary;
}

@media only screen and (min-width: 768px) {

  .header__icon {
    font-size: 3.2rem;
  }

  .header__title {
    font-size: 2.2rem;
  }
}
