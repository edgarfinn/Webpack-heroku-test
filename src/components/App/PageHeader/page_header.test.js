import PageHeader from './page_header';
import React from 'react';
import renderer from 'react-test-renderer';


test('header loads correctly', () => {
  const tree = renderer
    .create(<PageHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
