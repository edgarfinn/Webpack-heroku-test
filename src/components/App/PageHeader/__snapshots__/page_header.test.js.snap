// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header loads correctly 1`] = `
<header
  className="page__header"
>
  <i
    aria-hidden="true"
    className="header__icon fa fa-github-square"
  />
  <h1
    className="header__title"
  >
    Github Repo Finder
  </h1>
</header>
`;
